python/pyiceberg/table/snapshots.py

```diff
@@ -103,6 +110,12 @@ def __str__(self) -> str:
         result_str = f"{operation}id={self.snapshot_id}{parent_id}{schema_id}"
         return result_str
 
+    def fetch_manifest_list(self, io: FileIO) -> List[ManifestFile]:
```

Comment:
Minor: Should we add tests for this function?

Reply:
Good point @dhruv-pratap! Just added it 👍🏻